common-react/@dbeaver/ui-kit/src/Button/button.css

```diff
@@ -0,0 +1,164 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+@layer components {
+  .dbv-ui-button {
+    display: inline-flex;
+    text-align: center;
+    text-decoration: none;
+    cursor: pointer;
+    user-select: none;
+    position: relative;
+    gap: calc(var(--btn-padding-inline) / 4);
+    align-items: center;
+
+    border-radius: var(--btn-radius);
+    padding-inline: var(--btn-padding-inline);
+    padding-block: var(--btn-padding-block);
+
+    font-weight: var(--btn-font-weight);
+
+    &:disabled {
+      cursor: not-allowed;
+      opacity: 0.5;
+    }
+  }
+
+  .dbv-ui-button--small {
+    padding-inline: calc(var(--btn-padding-inline) * 0.5);
+    padding-block: calc(var(--btn-padding-block) * 0.5);
+    font-size: calc(var(--btn-font-size) * 0.875);
+    height: var(--control-height-small);
+  }
+
+  .dbv-ui-button--medium {
+    height: var(--control-height-medium);
+    font-size: var(--btn-font-size);
+  }
+
+  .dbv-ui-button--large {
+    padding-inline: calc(var(--btn-padding-inline) * 1.25);
+    padding-block: calc(var(--btn-padding-block) * 1.25);
+    font-size: calc(var(--btn-font-size) * 1.125);
+    height: var(--control-height-large);
+  }
+
+  .dbv-ui-button--xlarge {
+    padding-inline: calc(var(--btn-padding-inline) * 1.5);
+    padding-block: calc(var(--btn-padding-block) * 1.5);
+    font-size: calc(var(--btn-font-size) * 1.25);
+    height: var(--control-height-xlarge);
+  }
```

Comment:
why can't we adjust css variable instead, like:
```
  .dbv-ui-button--large {
    --btn-padding-inline: calc(var(--btn-padding-inline) * 1.25);
    --btn-padding-block: calc(var(--btn-padding-block) * 1.25);
    --btn-font-size: calc(var(--btn-font-size) * 1.125);
    --btn-height: var(--control-height-large);
  }
  ```
(this code may not work as expected but shows an idea)

Reply:
It also may be good to use a global multiplier (so you need to change only one variable)

Reply:
understood, working on that